docs/contributing/coding-style.md

```diff
@@ -264,6 +264,32 @@ func (s *something) SetSignedTok(st string) {
 }
 ```
 
+### Unused Variables
+
+It is an error to declare a variable without using it.
```

Comment:
I think the section `unused Variables` from the `test` section is better, can you move back to here?